### PR TITLE
LPS-169336 [Raylife AP] Filter bar creation

### PR DESCRIPTION
**Pull request by eliasantosf**
Task: [LPS-169336](https://issues.liferay.com/browse/LPS-169336)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx [hunk 1]
```diff
@@ -112,26 +235,18 @@ const ClaimsTable = () => {
 		{
 			greyColor: true,
 			key: 'claimStatus',
-			type: 'hasBubble',
+			type: 'status',
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx:238]: Should not this type be "hasBubble"?

**Review by joanagrego — CHANGES_REQUESTED** [review]
Good job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx (hunk 1 above)
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx]: Thaks, @joanagrego! Done!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/components/table/index.tsx [hunk 2]
```diff
@@ -55,6 +60,21 @@ const Table: React.FC<TableProps> = ({
...
+										onClick={setSortByDate}
+										symbol="order-arrow-down"
+									/>
+								) : (
+									<ClayButtonWithIcon
+										className="bg-neutral-0 btn-sm text-brand-primary-darken-1"
+										onClick={setSortByDate}
+										symbol="order-arrow-up"
+									/>
+								))}
```
[the remark this answers is not in the document]
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/components/table/index.tsx]: Thanks, @joanagrego! Done!

**Comment by eliasantosf** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]
Approved! Thank you @eliasantosf :white_check_mark:

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**